changelog/8509.improvement.rst

```diff
@@ -1,5 +1,5 @@
-Fixed issue where `TestCase.setUpClass` is not called when a test has `/` in its name since pytest 6.2.0.
+Fixed issue where :meth:`unittest.TestCase.setUpClass` is not called when a test has `/` in its name since pytest 6.2.0.
```

Comment:
```suggestion
Fixed issue where :meth:`unittest.TestCase.setUpClass` is not called when a test has ``/`` in its name since pytest 6.2.0.
```

Reply:
Btw, no need to worry about \` vs \`` because we configure the docs so they are equivalent:

https://github.com/pytest-dev/pytest/blob/501637547ecefa584db3793f71f1863da5ffc25f/doc/en/conf.py#L102-L103

Reply:
IMHO we should still be consistent though - right now we use an odd mix of both...

Reply:
Agree, just mentioning that is not necessary to go hunting for those, unless inclined to spend time on it for consistency reasons, because otherwise the end result is the same.